source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -216,9 +216,25 @@ def get_stop_area(self, obj):
         return StopAreaSerializer(obj).data
 
 
+feed_publisher_bano = {
```

Comment:
to be mutualized with the one from fields, no?

Reply:
done